lib/pbench/server/database/models/users.py

```diff
@@ -0,0 +1,121 @@
+import datetime
+from flask_bcrypt import generate_password_hash
+from sqlalchemy import Column, Integer, String, DateTime, LargeBinary
+from pbench.server.database.database import Database
+from sqlalchemy.orm import relationship
+
+
+class UserModel(Database.Base):
```

Comment:
I almost hate to even bring this up, but `UserModel` has always seemed a bit unwieldy, and it's the only SQL model object with the word "Model" in it... `User` might be better, but that's a trivial detail. Pretend I didn't say it.